backend/meshtalk/udp_transport.py

```
@@ -578,23 +632,36 @@ async def _maintenance_loop(self) -> None:
+            if not self._relays:
+                raise ConnectionError("TURN relay is not available")
+            self._relays[0].sendto(data, endpoint)
+            return
+        self._sendto(data, endpoint)
+
     def _sendto(self, data: bytes, endpoint: Endpoint) -> None:
         if not self._transport:
             raise RuntimeError("UDP transport is not started")
         self._transport.sendto(data, endpoint)
 
+    def _relay_datagram_received(self, data: bytes, addr: TurnEndpoint) -> None:
```

Ruff (0.16.2)

[error] Undefined name `TurnEndpoint`

(F821)